src/isIsogram.test.js

```diff
@@ -8,12 +8,25 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('1')).toBe('boolean');
   });
 
   it(`should be case sensitive`, () => {
+    expect(isIsogram(`MAtE`)).toBe(true);
+  });
```

Comment:
Here to check it you should have the same letter in uppercase and in lowercase (like `'Dad'` - should return `false` because `'D'` and `'d'` are the same letter for this task)